### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by fsenore** [Karry/osmscout-sailfish#272] [Q&A] [answered]
Hello, I hope that this is the correct way to contact you.
I am trying to use OSM Scout 2.16 for Sailfish OS on a Sony XA2 but I have a problem with tracks recording.
I tried recording some tracks but I did not see the track on the map. I also opened the Collections menu and I saw the recorded track entries, but I was unable to see them on the screen, even if I chose Show.

What am I doing wrong?

Thanks in advance.

Fulvio Senore

**Comment by Karry** [comment, marked as answer]
Hi Fulvio. Did you have downloaded some database for offline usage, and enable offline rendering in settings? OSM Scout is designed the way that it needs some offline database for most of its jobs. Without it, search, navigation and even displaying of collection entries is not working.

Lukas

**Comment by fsenore** [comment]
Hi Lukas,
thank you for your help. You are right, I did not install any offline map because they are really large.
Now I installed a map and now I can see the tracks.

It looks like the app shows all the tracks in the collection. It would be nice to be able to show single tracks only, to avoid clutter.

I downloaded the offline map for Italy and I noticed that there are no paths, only roads. Since I go hiking in the Alps paths are really needed. Do you know if I can download maps from OpenStreetMap that contain paths?

Fulvio

**Reply by Karry** [reply]
Paths should be there. In standard stylesheet, they are displayed from zoom level 15. But you can try to use "Outdoor" style (Map menu, offline map, style...). Outdoor style displays tracks from zoom level 14. Moreover, it display routes from zoom level 12. 

If you still don't see paths, even on higher zoom level, please share me the place with the problem. I may look to the OpenStreetMap data if there is some issue.

**Comment by fsenore** [comment]
You are right: I did not zoom in enough.
Even using the "Outdoor" style I need to zoom in a lot to see paths. Is there a way to see paths when zooming out a bit?

Fulvio
